SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -46,6 +46,7 @@ public IEnumerable<Transaction> FilterForBlock(IEnumerable<Transaction> transact
 
         public int MaxTxPerBlock => Settings.Default.MaxTransactionsPerBlock;
         public int MaxLowPriorityTxPerBlock => Settings.Default.MaxFreeTransactionsPerBlock;
+        public EnumSet<TransactionType> HighPriorityTxType => Settings.Default.HighPriorityTxType;
```

Comment:
Lets name the method name here something that doesn't indicate high priority since it is actually going to be a low priority transaction, just sorted higher than hashes. How about name it: `HigherLowPriorityTxTypes`

Reply:
Funny but true.